client/src/modules/patients/patients.service.js

```diff
@@ -22,23 +22,22 @@ PatientService.$inject = [
  *  }
  */
 function PatientService($http, util, Session, $uibModal,
-  Documents, Visits, Filters) {
-  var service = this;
+  Documents, Visits, Filters, AppCache, Periods, Api, $httpParamSerializer) {
   var baseUrl = '/patients/';
-  var filter = new Filters();
+  var service = new Api(baseUrl);
 
-  service.read = read;
+  var patientFilters = new Filters();
+  var filterCache = new AppCache('cash-filters');
```

Comment:
Oh - didn't catch this.  This should be `patient-filters` or something.  If you copied this code directly from the Cash Payments Registry, you will find it using Cash's filters.